website/migrations/0001_initial.py

```diff
@@ -6,7 +6,7 @@
 from django.db import migrations, models
 import django.db.models.deletion
 import website.models
```

Comment:
is this used?

Reply:
These are mostly autogenerated files (so that just always gets included I think) - best not to touch unless you have to! Which granted this particular extraction required customizing a few which was scary 😟

Reply:
i see- i just didn't see anything using website.models later on in the file!

Reply:
but i gotcha

Reply:
you are probably correct and it is an unnecessary import, but i just don't wan to get in the habit of editing these files